### ISSUE TITLE
Close all sets when stoping a protocol

### ISSUE THREAD

**Issue by pconesa** [scipion-em/scipion-em#250]
Otherwise other protocols might never start.

**Comment by dmaluenda** [comment]
Indeed, I think Scipion should take care of: if a protocol is ended (finished, aborted, failed....), it should have all the outputs closed. Maybe, before changing its status from running to any ended state is a good momemt to close all the outputs of that protocol.
